src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-wizard/rgw-multisite-wizard.component.html

```diff
@@ -0,0 +1,252 @@
+<div class="custom-modal-content">
+  <cd-modal [modalRef]="activeModal">
+    <ng-container i18n="form title"
+                  class="modal-title">Set up Multi-site Replication</ng-container>
+    <ng-container class="modal-content">
+      <div class="card">
+        <div class="container-fluid">
+          <cd-wizard [stepsTitle]="stepTitles"></cd-wizard>
+          <div class="card-body vertical-line">
+            <form [formGroup]="multisiteSetupForm"
+                  #formDir="ngForm"
+                  novalidate>
+              <ng-container [ngSwitch]="currentStep?.stepIndex">
+                <cd-alert-panel *ngIf="loading"
+                                spacingClass="mb-3"
+                                type="info">
+                  <span i18n>Please note that this process can take some time. During this period, do not click the back button or close the wizard. Thank you for your patience.</span>
```

Comment:
I don't think we need to Thank them for their patience. but somethings we need to ensure are
1. don't cancel the process if they accidentally close or come back from the modal
2. show the progress and logs so they know what is going on in the background.

I think the above 2 things will be certainly appreciated by them rather than annoyingly waiting for this to complete.

Reply:
@nizamial09  This was added for the time being when we didn't decide to show the progress of the setup. Since we'l be having the progress step now, this will be removed. Also this isn't going to take much long to complete in the real env. It should be roughly around 2-3 minutes and we'l be showing which step is going on in the progress page itself

Reply:
Also, we'l be having a separate PR for the progress step that am working on.